tests/hilti/types/function/hook-priorities.hlt

```diff
@@ -0,0 +1,42 @@
+# @TEST-EXEC: ${HILTIC} -j %INPUT foo.hlt bar.hlt >output
+# @TEST-EXEC: btest-diff output
+
+@TEST-START-FILE foo.hlt
+
+module Foo {
+
+import hilti;
+
+declare public hook void f();
+
+public function hook void f() { hilti::print("f with priority 0"); }
```

Comment:
Could make this `f with default priority` to decouple from whatever we use there.

Reply:
See above.